integrationTests/relayers/slowTests/framework/testSetup.go

```diff
@@ -26,6 +26,8 @@ const (
 	quorum                       = "03"
 )
 
+var addressToName = make(map[string]string)
```

Comment:
Please move this elsewhere. It is wrong to have "static" fields that can be overwritten by different tests. Maybe consider having this map as part of the KeysStore struct?